### PR TITLE
Implement createCloudConvertArchive function

### PR DESCRIPTION
**Pull request by geryjuhasz**

### CODE REVIEW

**Comment by geryjuhasz** [comment]
output
>

File: php/create-cloud-convert-archive/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# 📧 Archive files using CloudConverter's `createCloudConvertArchive` function
+A sample PHP Cloud Function for archiving files from AppWrite storage.
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:2]:
```suggestion
A sample PHP Cloud Function for archiving files from Appwrite storage.
```

File: php/create-cloud-convert-archive/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# 📧 Archive files using CloudConverter's `createCloudConvertArchive` function
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:1]:
```suggestion
# 📧 Archive files using CloudConvert's `createCloudConvertArchive` function
```

File: php/create-cloud-convert-archive/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+A sample PHP Cloud Function for archiving files from AppWrite storage.
+
+## 📝 Environment Variables
+Make sure to set the following environment variables.
+
+* **APPWRITE_ENDPOINT** - AppWrite Domain 
+* **APPWRITE_FUNCTION_PROJECT_ID** - AppWrite Project ID
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:8]:
```suggestion
```
You can remove this . the project ID is automatically passed to the function

File: php/create-cloud-convert-archive/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# 📧 Archive files using CloudConverter's `createCloudConvertArchive` function
+A sample PHP Cloud Function for archiving files from AppWrite storage.
+
+## 📝 Environment Variables
+Make sure to set the following environment variables.
+
+* **APPWRITE_ENDPOINT** - AppWrite Domain
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:7]:
```suggestion
* **APPWRITE_ENDPOINT** - Appwrite Domain 
```

File: php/create-cloud-convert-archive/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# 📧 Archive files using CloudConverter's `createCloudConvertArchive` function
+A sample PHP Cloud Function for archiving files from AppWrite storage.
+
+## 📝 Environment Variables
+Make sure to set the following environment variables.
+
+* **APPWRITE_ENDPOINT** - AppWrite Domain 
+* **APPWRITE_FUNCTION_PROJECT_ID** - AppWrite Project ID
+* **APPWRITE_API_KEY** - AppWrite API key
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:9]:
```suggestion
* **APPWRITE_API_KEY** - Appwrite API key
```

File: php/create-cloud-convert-archive/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+* Ensure that your folder structure looks like this 
+```
+.
+├── index.php
+├── vendor/
+├── composer.json
+└── composer.lock
+└── README.md
+```
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:30]: you're missing the instructions to package this function as a tar file. Please include those steps. You can use this as a reference https://github.com/appwrite/demos-for-functions/tree/master/php/get-covid-stats

File: php/create-cloud-convert-archive/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+## 🎯 Trigger
+
+```bash 
+php index.php fileId1 fileId2 ...
```
Comment (christyjacob4) [inline php/create-cloud-convert-archive/README.md:35]: This does not seem right.  You need to provide the triggers for this function like this https://github.com/appwrite/demos-for-functions/tree/master/php/get-covid-stats#-trigger

Here, you can also mention that you need to pass the files as part of the function data when triggering it 
```
path1 path2 path3  
```

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments and remove the `composer.lock` file

**Comment by christyjacob4** [comment]
@geryjuhasz closing this issue for now. Please feel free to reopen it/ create a new one when you're ready to start working on it once again

**Pull request closed without merging**